index.bs

```diff
@@ -1279,7 +1282,8 @@ The <dfn>context lost</dfn> steps for {{MLContext}} |context|, are:
 <summary>
 To <dfn for=MLContext>lose</dfn> {{MLContext}} |context| with {{DOMString}} |message|:
 </summary>
-    1. Let |info| be a new {{MLContextLostInfo}}.
+    1. Let |realm| be |context|'s [=relevant realm=].
+    1. Let |info| be a new {{MLContextLostInfo}} in |realm|.
```

Comment:
On re-reading, this is a dictionary and so also doesn't need the Realm. I think it's harmless, but to be consistent I removed it in aa9b161